telos-frontend/src/components/habit-tracker/JournalHabitTracker.css

```diff
@@ -1,89 +1,238 @@
-@charset "utf-8";
```

Comment:
The styling of the components is broken but this is likely related to #149

Reply:
Hmm, I don't know how to fix it. But I asked Harpreet to use his Mac laptop to test my code and which displays as expected.